src/main/java/tc/oc/pgm/api/setting/SettingValue.java

```diff
@@ -36,6 +36,9 @@
 
   SOUNDS_ON("sounds", "on"), // Always play sounds
   SOUNDS_OFF("sounds", "off"), // Never play sounds
+
+  VOTES_ON("vote-book", "on"), // Show the vote book on cycle
+  VOTES_OFF("vote-book", "off"), // Don't show the vote book on cycle
```

Comment:
Must be `vote_book`, look on line 58 in this class.

Reply:
https://github.com/Electroid/PGM/pull/315/files/d1cf7a0a2ff55e979f6ed0edfaec034eb368bbe3#diff-36e7b49e5ff5568811f0ef2f27fef7c1R58

Reply:
Good catch